### ISSUE TITLE
Facet for DataONE URL is broken

### ISSUE THREAD

**Issue by marisastrong** [CDLUC3/dash#71]
I was just poking around UCLA's Dash site and ran into an issue

 https://dash.ucla.edu/xtf/search?browse-all=yes

If I try clicking any of the links on the left-hand sidebar, there are
encoding errors in the URLs so the links don't work, e.g. clicking:

Publisher
•DataONE

Tries to load this URL -- note the "%2C%20dash-ucla.cdlib.org" in the
URL's path.

 http://dash.ucla.edu%2C%20dash-ucla.cdlib.org/xtf/search?f1-publisher=DataONE

Seems like this is the correct URL:

 https://dash.ucla.edu/xtf/search?f1-publisher=DataONE

**Comment by sfisher** [comment]
We had a superficially similar-looking problem in WAS—not sure if it's the same though.  In our case it was caused by the Apache server redirecting to the https version of the site and double-encoding when it redirected.  We added NE (no encode) on the redirect. http://stackoverflow.com/questions/6520484/mod-rewrite-urlencoding-an-already-urlencoded-querystring-parameter-any-way-to

But it may not be the same problem.  Just mentioning it in case it helps.
